substrate/primitives/consensus/babe/src/lib.rs

```diff
@@ -134,7 +134,7 @@ pub enum ConsensusLog {
 }
 
 /// Configuration data used by the BABE consensus engine.
-#[derive(Clone, PartialEq, Eq, Encode, Decode, RuntimeDebug)]
+#[derive(Clone, PartialEq, Eq, Encode, Decode, RuntimeDebug, TypeInfo)]
```

Comment:
Just osme random error I encountered: this type is used in a Runtime API and so must impl TypeInfo.